### PR TITLE
Tablet improvements

### PR DESCRIPTION
**Pull request by Alotor**
https://tree.taiga.io/project/penpot/us/1913?milestone=307334

### CODE REVIEW

File: frontend/src/app/main/data/workspace/drawing.cljs [hunk 1]
```diff
@@ -42,6 +42,15 @@
           (when (= tool :path)
             (rx/of (start-drawing :path)))
 
+          (when (= tool :curve)
+            (let [stopper (->> stream (rx/filter dwc/interrupt?))]
+              (->> stream
+                   (rx/take-until stopper)
```
Comment (niwinz) [inline frontend/src/app/main/data/workspace/drawing.cljs]: The stopper should be the last position.

File: frontend/src/app/main/data/workspace/selection.cljs [hunk 2]
```diff
@@ -47,51 +47,58 @@
...
+                    (update :x + (:x delta))
+                    (update :y + (:y delta)))
+
+                (-> selrect
+                    (update :width + (:x delta))
+                    (update :height + (:y delta)))))
+
+            selrect-stream
+            (->> ms/mouse-position
+                 (rx/take-until stoper)
```
Comment (niwinz) [inline frontend/src/app/main/data/workspace/selection.cljs]: The same here, stopper should be on the end of stream.

File: frontend/src/app/main/data/workspace/drawing.cljs (hunk 1 above)
Comment (Alotor) [reply frontend/src/app/main/data/workspace/drawing.cljs]: ok

File: frontend/src/app/main/data/workspace/selection.cljs (hunk 2 above)
Comment (Alotor) [reply frontend/src/app/main/data/workspace/selection.cljs]: ok

**Pull request merged**